v0/destinations/posthog/data/PHGroupConfig.json

```diff
@@ -8,8 +8,11 @@
     "required": false
   },
   {
-    "destKey": "groupId",
-    "sourceKeys": "groupId",
+    "destKey": "properties.$group_key",
+    "sourceKeys": [
+      "traits.groupKey", 
+      "groupId"
+    ],
```

Comment:
No need to map traits.groupKey to $group_key, just map groupId to $group_key.

Reply:
Updated.